### PR TITLE
Fix multiple sources for url for case studies, ensure fallback image …

### PR DESCRIPTION
**Pull request by helenb**
…is queried, and remove unused ref

### CODE REVIEW

File: src/components/process-block/process-block.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ class ProcessBlock extends React.Component {
     return (
       <div className={[styles.processBlock, className].join(' ')}>
         <div className={styles.processContainer}>
+          test what process block returns
```
Comment (chris-lawton) [inline src/components/process-block/process-block.js]: is this meant to be here?

Comment (helenb) [reply src/components/process-block/process-block.js]: Awkward! Will fix...

**Comment by helenb** [comment]
Sorted the test code. Is this ok otherwise @chris-lawton ?

**Comment by chris-lawton** [comment]
All good @helenb!

**Pull request merged**